publications/presentations/neup-meeting-02-03-21/slides.tex

```diff
@@ -0,0 +1,74 @@
+
+\begin{frame}
+\frametitle{Temoa}
+	\begin{itemize}
+		\item Minimizes the system cost
```

Comment:
Answer the question: what are we using Temoa for?

Reply:
We have a bit of this here already, just need to iron it out an rearrange some bits.